mlflow/tracking/client.py

```diff
@@ -5287,6 +5288,20 @@ def log_model_params(self, model_id: str, params: dict[str, str]) -> None:
             None
         """
         _validate_model_id_specified(model_id)
+        try:
+            self._tracking_client.get_logged_model(model_id)
+        except Exception as e:
+            reason = (
+                "the logged model may exist in a different workspace, "
+                if is_in_databricks_runtime()
+                else ""
+            )
+            raise MlflowException.invalid_parameter_value(
+                f"Failed to get logged model with ID {model_id}. "
+                f"You may not have access to the logged model, {reason}or the "
+                "logged model may have been deleted. If you are attempting to log parameters "
+                "to a Model Version, this is not supported."
+            ) from e
         return self._tracking_client.log_model_params(model_id, params)
```

Comment:
what does the current error message look like when the model is not found?

Reply:
On databricks it is now: `RestException: INVALID_PARAMETER_VALUE: Invalid model id: m-1234.` It's actually not accurate...